apps/src/aichat/views/ChatMessage.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styles from './chatMessage.module.scss';
+import classNames from 'classnames';
+
+interface ChatMessageProps {
+  id: string;
+  name: string;
+  role: string;
+  chatMessageText: string;
+  status: string;
+}
+
+const INAPPROPRIATE_MESSAGE = 'This message has been flagged as inappropriate.';
+const TOO_PERSONAL_MESSAGE = 'This message has been flagged as too personal.';
+
+const isAssistant = (role: string) => {
+  return role === 'assistant';
+};
+
+const isUser = (role: string) => {
+  return role === 'user';
+};
+
+const displayUserMessage = (status: string, chatMessageText: string) => {
+  if (status === 'ok') {
+    return (
+      <div
+        id={'chat-workspace-message-body'}
+        className={classNames(styles.message, styles.userMessage)}
+      >
+        {chatMessageText}
+      </div>
+    );
+  } else if (status === 'inappropriate') {
+    return (
+      <div
+        id={'chat-workspace-message-body-inappropriate'}
+        className={classNames(styles.message, styles.inappropriateMessage)}
+      >
+        {INAPPROPRIATE_MESSAGE}
+      </div>
+    );
+  } else if (status === 'personal') {
+    return (
+      <div
+        id={'chat-workspace-message-body-too-personal'}
+        className={classNames(styles.message, styles.tooPersonalMessage)}
+      >
+        {TOO_PERSONAL_MESSAGE}
+      </div>
+    );
+  }
```

Comment:
Unsure if I need an `else` to return a `<div></div>`.

Reply:
It [seems](https://react.dev/learn/conditional-rendering#conditionally-returning-nothing-with-null) like `null` is appropriate?